### PR TITLE
Add RDNAPTRANS™2018 grids for The Netherlands.

### PR DESCRIPTION
**Pull request by sebastic**
The old RDNAPTRANS™2008 grids for The Netherlands (`rdtrans2008.gsb` & `naptrans2008.gtx`) are currently used in PROJ, but could not be included in proj-datumgrid due to a non-free license. Details available in: [Use of RDTRANS2008 and NAPTRANS2008.pdf ](https://salsa.debian.org/debian-gis-team/proj-rdnap/raw/upstream/2008+2018/Use%20of%20RDTRANS2008%20and%20NAPTRANS2008.pdf)

The new RDNAPTRANS™2018 grids use the CC-BY-4.0 license, and come in two variants. Details available in: [RDNAPTRANS2018.pdf](https://salsa.debian.org/debian-gis-team/proj-rdnap/raw/upstream/2008+2018/RDNAPTRANS2018.pdf)

EPSG 9.8.6 now includes changes for RDNAPTRANS™2018:

 * EPSG:9281 uses 7 parameter transformation
 * EPSG:9282 uses `rdtrans2018.gsb` (variant 2) for the horizontal transformation
 * EPSG:9283 uses `nlgeo2018.gtx` (variant 1) for the vertical transformation.

There are still some outstanding issues which have prevented the inclusion of `rdcorr2018.gsb` (variant 1) in EPSG, but it's expected in a future EPSG update. For now the combination of variants needs to used (starting 1 October 2020 when the 2008 should no longer be used), instead of only the recommended variant 1.

I'm not sure if the changes to `filelist.csv` are correct.

CC: @Jochem-L

### CODE REVIEW

File: filelist.csv [hunk 1]
```diff
@@ -232,3 +232,7 @@ ontpht1964-nzvd2016.gtx,VERTICAL_OFFSET_VERTICAL_TO_VERTICAL,New Zealand - One T
...
 taraht1970-nzvd2016.gtx,VERTICAL_OFFSET_VERTICAL_TO_VERTICAL,New Zealand - Taranaki,,EPSG:7839,EPSG:5769,EPSG:4167,nz_linz,Land Information New Zealand,Creative Commons Attribution 4.0 International licence https://creativecommons/licenses/by/4.0/
 wellht1953-nzvd2016.gtx,VERTICAL_OFFSET_VERTICAL_TO_VERTICAL,New Zealand - Wellington,,EPSG:7839,EPSG:5770,EPSG:4167,nz_linz,Land Information New Zealand,Creative Commons Attribution 4.0 International licence https://creativecommons/licenses/by/4.0/
+rdcorr2018.gsb,HORIZONTAL_OFFSET,Netherlands,,EPSG:4289,EPSG:4258,,nl_nsgi,Nederlandse Samenwerking Geodetische Infrastructuur (NSGI),Creative Commons Attribution 4.0 International licence https://creativecommons/licenses/by/4.0/
+nlgeo2018.gtx,VERTICAL_OFFSET_VERTICAL_TO_VERTICAL,Netherlands,,EPSG:4937,EPSG:5709,,nl_nsgi,Nederlandse Samenwerking Geodetische Infrastructuur (NSGI),Creative Commons Attribution 4.0 International licence https://creativecommons/licenses/by/4.0/
```
Comment (rouault) [inline filelist.csv]:
```suggestion
nlgeo2018.gtx,VERTICAL_OFFSET_GEOGRAPHIC_TO_VERTICAL,Netherlands,,EPSG:4937,EPSG:5709,,nl_nsgi,Nederlandse Samenwerking Geodetische Infrastructuur (NSGI),Creative Commons Attribution 4.0 International licence https://creativecommons/licenses/by/4.0/
```

File: filelist.csv [hunk 2]
```diff
@@ -232,3 +232,7 @@ ontpht1964-nzvd2016.gtx,VERTICAL_OFFSET_VERTICAL_TO_VERTICAL,New Zealand - One T
...
+rdtrans2018.gsb,HORIZONTAL_OFFSET,Netherlands,,EPSG:4289,EPSG:4258,,nl_nsgi,Nederlandse Samenwerking Geodetische Infrastructuur (NSGI),Creative Commons Attribution 4.0 International licence https://creativecommons/licenses/by/4.0/
+naptrans2018.gtx,VERTICAL_OFFSET_VERTICAL_TO_VERTICAL,Netherlands,,EPSG:4937,EPSG:5709,,nl_nsgi,Nederlandse Samenwerking Geodetische Infrastructuur (NSGI),Creative Commons Attribution 4.0 International licence https://creativecommons/licenses/by/4.0/
```
Comment (rouault) [inline filelist.csv]:
```suggestion
naptrans2018.gtx,VERTICAL_OFFSET_GEOGRAPHIC_TO_VERTICAL,Netherlands,,EPSG:4937,EPSG:5709,,nl_nsgi,Nederlandse Samenwerking Geodetische Infrastructuur (NSGI),Creative Commons Attribution 4.0 International licence https://creativecommons/licenses/by/4.0/
```

File: europe/README.EUROPE [hunk 3]
```diff
@@ -106,6 +106,28 @@ The Malin Head datum is used in the Republic of Ireland.
...
+* rdcorr2018.gsb
+* nlgeo2018.gtx
+
+Recommended variant, applies the datum transformation as a separate step using
+a 3D similarity transformation.
+
+Variant 2:
+
+* rdtrans2018.gsb
+* naptrans2018.gtx
```
Comment (rouault) [inline europe/README.EUROPE]: isn't there an inversion between nlgeo2018.gtx and naptrans2018.gtx ? I see EPSG has only entries for rdtrans2018.gsb + nlgeo2018.gtx ? Or is EPSG shipping with a mix of current variant 1 & variant 2

Comment (sebastic) [reply europe/README.EUROPE]: Yes, unfortunately EPSG 9.8.6 is shipping a mix of variant 1 & 2, as mentioned in the OP.

I've included all 4 grids in this PR to not have to add them later once EPSG is updated, even though EPSG only uses a subset of the grids at time of writing.

Comment (rouault) [reply europe/README.EUROPE]: > Or is EPSG shipping with a mix of current variant 1 & variant 2

Ah my bad, you mentionned that in your initial message. From a meeting that I attended, I know EPSG is not a big fan of the oddities of RDNAPTRANS ;-)

**Comment by rouault** [comment]
Perhaps one or two of the above mentionned PDF could be referenced in the description of README.EUROPE ?

**Comment by sebastic** [comment]
> Perhaps one or two of the above mentioned PDF could be referenced in the description of README.EUROPE ?

Yes, RDNAPTRANS2018.pdf should be consulted by users, as it's the best documentation for the grids to date.

I'd rather not link to the Debian package git repo, but I guess we don't have much of a choice. grid_alternatives already does it too because the files are not easily available at the authoritative source.

**Review by rouault — APPROVED** [review]

**Comment by rouault** [comment]
@sebastic Willing to submit the corresponding PR to proj itself to register the 2 grids used by EPSG in grid_alternatives ?

**Comment by Jochem-L** [comment]
Variant 1 for the horizontal transformation will not be included in EPSG.

EPSG:9282 uses rdtrans2018.gsb (variant 2) for the horizontal transformation. A disadvantage is that it can not be used outside the grid bounds. When outside the grid bounds, EPSG:9281 could be used to obtain the same results as Variant 1.

EPSG:9281 uses only a 7 parameter transformation, which gives an approximated transformation with an accuracy of 25 cm.

**Pull request merged**